src/main/java/eu/evops/maven/plugins/cucumber/parallel/Run.java

```diff
@@ -19,12 +19,7 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Properties;
+import java.util.*;
```

Comment:
Can you expand the wildcard import? I have learnt over the years it is not a good practice as it pollutes your namespace. Reasonable explanation can be found at https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad.

Reply:
Sure thing.